README.md

```diff
@@ -15,33 +15,36 @@
 
 All of these tools must be run on the same host as ClickHouse server is running.
 
-## Local development (using venv)
+## Local development (using poetry)
 
 ```sh
-python3 -m venv venv
-source venv/bin/activate
-pip install .[test]
-flit build --no-use-vcs
-flit install
+sudo make install-poetry
+# or to install in user's home directory
+make POETRY_HOME=~/opt/poetry install-poetry
+
+# For building deb packages
+sudo make prepare-build-deb
+sudo make build-deb-package  
```

Comment:
I think it's better to place instructions on building deb package after ones on running tests.

And it would be great if these commands work without `sudo`.

Reply:
I did it. And yes, now it can work without sudo.